### DISCUSSION TITLE
about the screensaver pattern

### DISCUSSION THREAD

**Discussion by saltat** [BOINC/boinc#5980] [Q&A]
### Describe the bug

I have let it run only in the night, but I do not want to open it everyday, so I did not quit it during the daytime. I do not want it to show its screen saving patterns when I'm not using my computer, because it is really annoying to see a screen saving started when I'm reading or watching something on my computer. How can I stop it?

**Comment by AenBleidd** [comment]
I don't believe there is any OS that let you set different screensaver settings for day and night.
BOINC just uses OS built-in functionality to show screensaver, so there is nothing customized present.
I suggest to ask this question to your OS provider.

**Reply by saltat** [reply]
yeah, so I just want to close all screen savers. I have already locked it in my OS settings, but BOINC is still showing its.

**Reply by NenTech** [reply]
Which OS are you using? Otherwise use the put screen to sleep option.

**Comment by CharlieFenton** [comment]
BOINC does not need the screensaver to do its work. Just turn off the screensaver but leave BOINC itself set up as you already have it.
